requirements_devel.txt

```diff
@@ -7,5 +7,72 @@
 
 # Development packages
 
-django-debug-toolbar
+
 coverage
+alabaster==0.7.10
```

Comment:
on second thought.. i'm merging a branch with the cleaned up requirements. you can update your branch with develop once that's done